Paralayout.podspec

```diff
@@ -8,8 +8,4 @@ Pod::Spec.new do |s|
   s.source   = { :git => 'https://github.com/square/Paralayout.git', :tag => s.version }
   s.source_files = 'Paralayout/*.{swift}'
   s.ios.deployment_target = '13.0'
-
-  s.test_spec 'Tests' do |test_spec|
-    test_spec.source_files = 'ParalayoutTests/*{.swift}'
-  end
```

Comment:
Testing `ParalayoutTests` via Cocoapods broke with latest Xcode? Or at least I couldn't get it to work. Was failing on `rsync`, which [we need to enable script sandboxing](https://stackoverflow.com/questions/76590131/error-while-build-ios-app-in-xcode-sandbox-rsync-samba-13105-deny1-file-w), which I don't know how to do in the generated podspec workspace.

The good news is our `Package.swift` is running these tests – we don't need to run them twice. IMO it's a good practice these days to keep Cocoapods as a distribution mechanism and not use it for anything else.

Reply:
okay, this wasn't the issue. even with this gone we're still getting the failure. uh... @dnkoutso any chance you've got an idea here? I have zero clue why Cocoapods is executing a script outside of sandbox on Paralayout but not, say, Valet.

Reply:
The actual errors:
```
    error: Sandbox: rsync.samba(3367) deny(1) file-write-create /Users/dfed/Library/Developer/Xcode/DerivedData/App-dxcglffjzvhcjiawuhggckjmaofx/Build/Products/Release-iphonesimulator/App.app/Frameworks/Paralayout.framework/_CodeSignature (in target 'App' from project 'App')
    error: Sandbox: rsync.samba(3368) deny(1) file-write-create /Users/dfed/Library/Developer/Xcode/DerivedData/App-dxcglffjzvhcjiawuhggckjmaofx/Build/Products/Release-iphonesimulator/App.app/Frameworks/Paralayout.framework/.Info.plist.yOJmBZ (in target 'App' from project 'App')
    error: Sandbox: rsync.samba(3368) deny(1) file-write-create /Users/dfed/Library/Developer/Xcode/DerivedData/App-dxcglffjzvhcjiawuhggckjmaofx/Build/Products/Release-iphonesimulator/App.app/Frameworks/Paralayout.framework/.Paralayout.UithR4 (in target 'App' from project 'App')
```

Reply:
oh heck this looks like https://github.com/CocoaPods/CocoaPods/issues/12664

Reply:
fixed it by bumping CP again.